src/SAML2/SignedElementHelper.php

```diff
@@ -15,54 +16,41 @@
  *
  * @package SimpleSAMLphp
  */
-class SignedElementHelper extends SignedElement
+trait SignedElementHelper
 {
     /**
-     * Available methods for validating this message.
-     *
-     * @var array
-     */
-    private $validators = [];
-
-    /**
-     * How long this element is valid, as a unix timestamp.
+     * SignedElement constructor.
```

Comment:
This is not a constructor?

Reply:
Good catch! I don't know what happened here...

Reply:
Lack of coffee is what happened :P

Reply:
Amen, brother!